### ISSUE TITLE
Issues with the .bat file

### ISSUE THREAD

**Issue by OrsonDeWitt** [wleepang/DesktopDeployR#47]
1. In task manager, the task is called "R for Windows front-end". Is there a way to rename it?
2. After attempting to load data into the app and closing it before completion, the app breaks and does not allow to access it again until it finishes loading the data (in case of an error this could take hours). Running the script again does not open another instance of the app. Is there a way to circumvent such a case? 
3. I have the same exact issue as #19  on another laptop. It does not open, and if I run it as administrator, I get the wsj error. I have 
` session$onSessionEnded(function() {stopApp()}) `
Included, and from what I understand the need to set path with cd has been fixed. So I don't understand what's causing the issue.

**Comment by nev-awaken** [comment]
Try putting session$onSessionEnded(function() {stopApp()}) inside the server function.

**Comment by nev-awaken** [comment]
Is in your config file the paths are same as below?

Defaults to:
				"dist\\R-Portable\\App\\R-Portable\\bin\\"

			To use a system installed R set to something like:
				"dist\\R-Portable\\App\\R-Portable\\bin\\"
				
				
				
				"r_exec": {
		"home": "dist\\R-Portable\\App\\R-Portable\\bin\\"
	},

**Comment by OrsonDeWitt** [comment]
> Is in your config file the paths are same as below?
> 
> Defaults to: "dist\R-Portable\App\R-Portable\bin\"
> 
> ```
> 		To use a system installed R set to something like:
> 			"dist\\R-Portable\\App\\R-Portable\\bin\\"
> 			
> 			
> 			
> 			"r_exec": {
> 	"home": "dist\\R-Portable\\App\\R-Portable\\bin\\"
> },
> ```

Yes, they are; I have actually figured the "run.wsf" part since I posted this, thank you :) the problem was that the laptop that I was I testing it on had non-Latin characters in the computer name. Once we tried on another drive, it worked.

**Comment by nev-awaken** [comment]
Did you figure out the "R for front end" part?
